cms/djangoapps/contentstore/views/course.py

```diff
@@ -1249,6 +1250,15 @@ def advanced_settings_handler(request, course_key_string):
 
                         # now update mongo
                         modulestore().update_item(course_module, request.user.id)
+                        # Throw away old versions of CourseOverview and CourseOverviewTab - added by labster
+                        # as they might contain stale data then create new one.
+                        try:
+                            course_overview = CourseOverview.objects.get(id=course_key)
+                            CourseOverviewTab.objects.filter(course_overview=course_overview).delete()
+                            course_overview.delete()
+                        except CourseOverview.DoesNotExist:
+                            pass
+                        CourseOverview.load_from_module_store(course_key)
```

Comment:
Delete old data of CourseOverview and CourseOverviewTab then
Create new one CourseOverview from data course module that has been updated so when user try to activate a voucher will process with up to date Data/settings(ex. invitation_only).